testkit-core/src/main/scala/org/bitcoins/testkitcore/util/BaseAsyncTest.scala

```diff
@@ -30,8 +30,7 @@ trait BaseAsyncTest
     with BeforeAndAfterAll
     with Matchers
     with ScalaCheckPropertyChecks
-    with AsyncTimeLimitedTests
-    with BitcoinSLogger { this: AsyncTestSuite =>
+    with AsyncTimeLimitedTests { this: AsyncTestSuite =>
```

Comment:
Is this still needed to remove `BitcoinSLogger` since the trait is not used in `cryptoTest`?